problem.schema.json

```diff
@@ -0,0 +1,136 @@
+                    "type": {
+                        "description": "Type of the variable.",
+                        "enum": [
+                            "CONTINUOUS",
+                            "BINARY",
+                            "INTEGER"
+                        ]
+                    }
+                 }
+            }
+        },
+        "constraints": {
+            "description": "An array containing the constraints of the problem.",	
+            "type": "array",
+            "items":  {
+                "type": "object",
+                  "required": ["type", "lhs", "rhs"],
+                 "properties": {
+                    "name": {
+                        "description": "Name of the constrainn.",
+                        "type": "string"
+                    },
+                    "type": {
+                        "description": "Type of the inequality.",
+                        "enum": [
+                            "EQ",
+                            "LEQ",
+                            "GEQ"
+                        ]
+                    },
+                    "lhs": {
+                        "description": "An array containing the terms in the left-hand side of the constraints of the problem.",
+                        "type": "array",
+                        "items":  {
+                            "type": "object",
+                              "required": ["name", "stage", "coefficient"],
+                              "properties": {
+                                "name": {
+                                    "description": "Name of the variable.",
+                                    "type": "string"
+                                },
+                                "stage": {
+                                    "description": "Problem stage which the variable occurs.",
+                                    "type": "integer",
+                                    "minimum": 0
+                                },
+                                "coefficient": {
+                                    "description": "Constraint coefficient of the variable.",
+                                    "$ref": "#/definitions/AffineFunction"
+                                }
+                            }
+                        }
+                    },
+                    "rhs": {
+                        "description": "An array containing the terms in the right-hand side of the constraints of the problem.",
+                        "$ref": "#/definitions/AffineFunction"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "AffineFunction": {
+            "type": "array",
+            "items": {
+                "oneOf": [{
+                    "description": "a coefficient",
+                    "type": "number"
+                }, {
+                    "enum": [
+                        "inf",
+                        "-inf"
+                    ]
```

Comment:
What other strings are allowed? Can I write `"ub": ["demand"]` where `demand` is the name of a random variable?